tests/test_wrdcld.py

```diff
@@ -0,0 +1,42 @@
+from string import ascii_letters, digits, punctuation
+from unittest import TestCase
+
+import hypothesis.strategies as st
+from hypothesis import given
+
+from wrdcld import make_word_cloud
+
+
+@st.composite
+def words_with_repeats_strategy(draw):
+    words = draw(
+        st.lists(st.text(ascii_letters + digits + punctuation, min_size=1), min_size=1)
+    )
+
+    frequencies = draw(
```

Comment:
ℹ️ This is nice - just a thought from reading this: It might be useful to also support supplying an already-created `Counter` object, especially if people have already processed _huge_ lists of words.

Reply:
good idea, I'll make an issue